wire-ios-request-strategy/Sources/Request Strategies/Connection/ConnectionRequestStrategyTests.swift

```diff
@@ -247,6 +251,33 @@ class ConnectionRequestStrategyTests: MessagingTestBase {
         }
     }
 
+    func testThatOneOnOneResolverIsInvoked_WhenConnectionRequestIsAccepted() throws {
```

Comment:
Perhaps we can also test the delay by increasing it longer than the timeout and asserting the failure.

Reply:
Done!